CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ option(BUILD_SHARED_LIBS "Build dynamic libraries" ON)
 
 # Override default CXX compile/link lines for Windows after project
 if(WIN32)
-  set(CMAKE_CXX_STANDARD 14)
```

Comment:
Could you please check if we can remove hardcoded C++ standard version here completely? As I know we don't need it on Linux, so I wonder if we can do the same for Windows.

Reply:
sure, removed, tests are still passing

Reply:
Thank you!